spec/helpers_spec.cr

```diff
@@ -12,7 +12,7 @@ require "../src/invidious/search"
 require "../src/invidious/trending"
 require "../src/invidious/users"
 
-CONFIG = Config.from_yaml(File.open("config/config.yml"))
+CONFIG = Config.from_yaml(File.open("config/config.example.yml"))
```

Comment:
What is the point of this change? I guess if you want to install invidious you have to copy config.example.yml to config.yml?

Reply:
> So, if I understand correctly: it's the same as #1428 except it doesn't break CI, will work with existing instances, and doesn't contain every options (see [c0aca18](https://github.com/iv-org/invidious/commit/c0aca1883c81cd3ebed14f5cee6d0eb8e834d862) )?
> 
> Basically we just need to add add everything to config.example.yml and it will work?

Yes, correct.

> How will docker environment variable work with that though?

It is unaffected, it will work the same as before.

Reply:
okay, I get it. spec contains the CI test.